src/components/util/useStyles.ts

```diff
@@ -26,6 +26,30 @@ const useStyles = makeStyles(theme => ({
 	'paddedCaption': {
 		padding: theme.spacing(2, 0, 2, 0)
 	},
+	'eventContainer': {
+		display: 'flex', justifyContent: 'center'
+	},
+	'eventPaper': {
+		height: '98vh', width: '70vh', alignSelf: 'center', overflow: 'auto'
+	},
+	'iconMargin': {
+		margin: theme.spacing(0, 1, 0, 0)
+	},
+	'eventInnerContainer': {
+		margin: theme.spacing(2, 0),
+		display: 'flex',
+		flexDirection: 'column'
+	},
+	'eventButton': {
+		alignSelf: 'center',
+		margin: 2
+	},
+	'eventOptionText': {
+		margin: theme.spacing(3, 0)
+	},
+	'eventGrid': {
+		margin: theme.spacing(2, 0, 0, 0)
+	},
```

Comment:
So this `useStyles` file is technical debt from when I first started learning Material UI - we should get rid of it but this will be done in a later PR instead. For now, could you move these new rules you've created into the file they're used in? You can do `makeStyles` in that file. Have a look at the registration page for an example of how to do this.